connector/queue/worker.py

```diff
@@ -263,6 +263,8 @@ def available_db_names():
             db_names = db.exp_list(True)
         dbfilter = config['dbfilter']
         if dbfilter and db_names:
+            # Remove %d and %h from dbfilter (issue #58)
+            dbfilter = re.sub('(%d|%h).*$', '', dbfilter)
             db_names = [d for d in db_names if re.match(dbfilter, d)]
```

Comment:
Does it make sense to match on dbfilter after removing %d and %h?

As @guewen noted in #58 it probably does not make sence to use dbfilter at all in the job runner.

So we could completely remove this filtering, or to be less brutal towards user who would depend on this, skip the filtering if dbfilter contains %d or %h.